lib/injectable/version.rb

```diff
@@ -1,3 +1,3 @@
 module Injectable
-  VERSION = "0.1.0"
+  VERSION = '1.0.0'
```

Comment:
When you change the version you have to update the lockfile as well, run `bundle install`

Reply:
Good to know, thanks!